apiserver/params/network.go

```diff
@@ -830,6 +830,12 @@ type NetworkInfoParams struct {
 	Endpoints  []string `json:"bindings"`
 }
 
+// UpdateNetworkInfoParams refreshes the network information for the bound
+// endpoints for each one of the requested unit names.
+type UpdateNetworkInfoParams struct {
```

Comment:
So this can be deleted now right?

Reply:
Doh! sorry; missed that. It has now been removed.